tests/on_target/tests/test_ppk/test_power.py

```diff
@@ -158,9 +159,13 @@ def thingy91x_ppk2():
     recover_device()
     ppk2_dev.stop_measuring()
 
-
-@pytest.mark.dut_ppk
+@pytest.mark.slow
 def test_power(thingy91x_ppk2, hex_file):
+    '''
+    Test that the device can reach PSM and measure the current consumption
+
+    Current consumption is measured and report generated.
+    '''
     flash_device(os.path.abspath(hex_file))
```

Comment:
this wont work also I think, now it will try to flash ppk dut by using SEGGER env, but SEGGER env is pointing to dut1.
Can be solved with:
`flash_device(os.path.abspath(hex_file), serial=SEGGER_PPK)`
same thing also for reset and recover